### PR TITLE
Setup for reviews module

### PR DESCRIPTION
**Pull request by ArohanD**
Setup environment, including DB seed, DB link, server, and react.

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
+const express = require('express')
```
Comment (jxkim) [inline server/server.js]: Missing semicolon

File: server/server.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+const app = express()
```
Comment (jxkim) [inline server/server.js]: Missing semicolons (lines 1-2)

File: server/server.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+const port = 3333;
+
+const bodyParser = require('body-parser')
```
Comment (jxkim) [inline server/server.js]: Per Jarrod, group external modules together at top

File: server/server.js [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+  let hipsum = `this text is from the server`
+  let helpful = Math.floor((Math.random() * 32) + 1)
+  var singleItem = new Item({
+      reviewID: 7777,
+      author: 'testAuth',
+      rating: rating,
+      title: 'ABRA KADABRA ALAKAZAM',
+      text: hipsum,
+      helpfulCount: helpful
+  });
```
Comment (jxkim) [inline server/server.js]: If possible, create helper function for this in separate file

File: server/server.js [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+      let helpful = Math.floor((Math.random() * 32) + 1)
+      var singleItem = new Item({
+          reviewID: i,
+          itemID: itemID,
+          author: 'testAuth',
+          rating: rating,
+          title: 'SHAZAM KABLAM ALAKAZAM',
+          text: hipsum,
+          helpfulCount: helpful
+
```
Comment (jxkim) [inline server/server.js]: Might also consider extracting this into another file

File: server/server.js (hunk 2 above)
Comment (ArohanD) [reply server/server.js]: updated

File: server/server.js (hunk 3 above)
Comment (ArohanD) [reply server/server.js]: updated

File: server/server.js (hunk 4 above)
Comment (ArohanD) [reply server/server.js]: will do later

File: server/server.js (hunk 5 above)
Comment (ArohanD) [reply server/server.js]: considered

File: DataEntryHelper.js [hunk 6]
```diff
@@ -0,0 +1,40 @@
+
+//NOTE: Change 'lamps' to your category
+const categoryFolder = 'lamps';
+// const printArr  = (folder, num) => {
```
Comment (wiggitywhitney) [inline DataEntryHelper.js]: Learn App recommends we remove our code that's been commented out

File: README.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+
+Here are the user stories explored to create this app:
+
+- [ ] As a user, I should be able to see aggregate ratings for the item I am exploring.
+- [ ] As a user, I should be ble to read reviews relevant to the item I am exploring.
+  - [ ] As a user, I should be able to view images attached to these reviews.
+- [ ] As a user, I should be able to write a review, and have the surrounding components update accordingly.
+  - [ ] As a user, I should also be able to submit photos to this review.
+- [ ] As a user, I should be able to mark reviews as helpful.
+
```
Comment (wiggitywhitney) [inline README.md]: watch your spacing (also good work starting your README)

File: dist/styles.css [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+  width: 300px;
+  border-bottom: 1px solid grey;
+  padding-bottom: 20px;
+}
+
+#rev_overall_stars {
+  width: 50%;
+}
+
+
```
Comment (wiggitywhitney) [inline dist/styles.css]: one return too many

**Pull request merged**